source/index.ts

```diff
@@ -136,6 +136,7 @@ namespace is { // tslint:disable-line:no-namespace
 	export const nullOrUndefined = (value: any): value is null | undefined => null_(value) || undefined(value);
 	export const object = (value: any): value is object => !nullOrUndefined(value) && (function_(value) || isObject(value));
 	export const iterable = (value: any): value is IterableIterator<any> => !nullOrUndefined(value) && function_(value[Symbol.iterator]);
+	export const asyncIterable = (value: any): value is AsyncIterableIterator<any> => !nullOrUndefined(Symbol.asyncIterator) && function_(value[Symbol.asyncIterator]);
```

Comment:
You also need the `!nullOrUndefined(value)` check, like in the above `iterable` predicate.

It would currently fail on `t.false(m.asyncIterable(undefined));`